code/modules/mob/living/carbon/carbon.dm

```diff
@@ -211,7 +211,12 @@
 			AdjustKnockedout(-3)
 			AdjustStunned(-3)
 			AdjustKnockeddown(-3)
-
+			if(halloss > 0)
+				if(ishuman(M))
```

Comment:
Since this affects only humans you'll want it under `/mob/living/carbon/human/proc/handle_regular_status_updates()`

Reply:
I'm forward thinking on this in case we have antags/others that halloss interactions might apply to in the future.

Reply:
They are more likely to be a subset of humans, such as synths are...